src/components/unique-token/UniqueTokenAttributeItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { UniqueAssetTrait } from '../../entities/uniqueAssets';
+import Tag from '../Tag';
+import transformUniqueAssetTraitsForPresentation from '@rainbow-me/helpers/transformUniqueAssetTraitsForPresentation';
+import { magicMemo } from '@rainbow-me/utils';
+
+type AttributeItemProps = UniqueAssetTrait &
+  ReturnType<typeof transformUniqueAssetTraitsForPresentation> & {
+    color: string;
+    slug: string;
+    max_value?: string | number;
+  };
```

Comment:
Should I extract the type in a better way instead of using ReturnType TS helper? If so where should I put this type? What's your convention?

Reply:
i don't mind